docs/source/api_reference.rst

```diff
@@ -28,7 +28,10 @@ API Reference
    aihwkit.nn.modules.container
    aihwkit.nn.modules.conv
    aihwkit.nn.modules.linear
-   aihwkit.nn.modules.lstm
+   aihwkit.nn.modules.rnn
+   aihwkit.nn.modules.rnn.cells
+   aihwkit.nn.modules.rnn.layers
+   aihwkit.nn.modules.rnn.rnn
```

Comment:
Non-blocking - originally, this file meant to contain only the Python _packages_ (ie. the directories) instead of all the modules (ie. the single `.py` files). The idea was to provide an overview rather than an exhaustive list of all modules, with the thinking that users could navigate to each package for more information about its modules (avoiding having a lot of cruft in this file, for example the `cloud` internals that are rarely used directly).

There are some exceptions to that policy already in this file in `master` (maybe due to some modules having particular importance) - but I wonder if it would make sense to just include `aihwkit.modules.rnn` in this case (and/or take a chance to unify, as for example `aihwkit.nn.modules.conv_mapped` and `linear_mapped` are missing, making it a bit inconsistent).